src/config.js

```diff
@@ -230,6 +230,31 @@ const baseConfig = {
       }
     }
   },
+  oidc: {
+    entraId: {
+      oidcWellKnownConfigurationUrl: {
+        doc: 'Entra OIDC .well-known configuration URL',
+        format: String,
+        env: 'ENTRA_OIDC_WELL_KNOWN_CONFIGURATION_URL',
+        default:
+          'https://login.microsoftonline.com/6f504113-6b64-43f2-ade9-242e05780007/v2.0/.well-known/openid-configuration'
+      },
+      audience: {
+        doc: 'Admin UI app as audience',
+        format: String,
+        env: 'ADMIN_UI_AS_AUDIENCE',
```

Comment:
I think the name `ADMIN_UI_ENTRA_CLIENT_ID` would better describe what value this env var should have (when looking in `cdp-app-config`)